.circleci/crashed.txt

```diff
@@ -0,0 +1,7 @@
+*test_softmax_axis_0*
```

Comment:
would it be better to keep under onnxifi_driver_test/crashed.txt ?

Reply:
Like .circleci/onnxifi_driver_test/crashed.txt?  Or something else?

Reply:
yeah, that sounds good.